lib/wab/controller.rb

```diff
@@ -231,7 +231,7 @@ def form_where_eq(key, value)
         else
           x << "'" + value
         end
-      elsif '2' == RbConfig::CONFIG['MAJOR'] && '4' > RbConfig::CONFIG['MINOR'] && Fixnum == value_class
+      elsif 24 > WAB::Utils.RUBY_SERIES && Fixnum == value_class
```

Comment:
Lets take it a step further. Note that in all case the checks are being made on the class of a value. How about a method that returns true if the value class is Integer or Fixnum but only check Fixnum if Ruby 2.3 to avoid the warning in 2.4?

Oh, I believe it is Ruby convention that all uppercase values are usually constants and not methods. Thats a non-issue if you put an ```is_int?``` or ```is_i?``` in the utilities.